packages/hash/frontend/src/pages/[account-slug]/types/entity-type/tab-link.tsx

```diff
@@ -0,0 +1,78 @@
+import { frontendUrl } from "@hashintel/hash-shared/environment";
+import { Box, Stack, Tab, Typography } from "@mui/material";
+import { useRouter } from "next/router";
+import { FunctionComponent } from "react";
+
+export type TabLinkProps = {
+  label: string;
+  value: string;
+  count?: number;
+};
+
+export const TabLink: FunctionComponent<TabLinkProps> = ({
+  label,
+  value,
+  count,
+  ...props
+}) => {
+  const router = useRouter();
+
+  const active = `${frontendUrl}${router.asPath}` === value;
```

Comment:
this component would be reusable except for this – it also makes it a bit harder to update the format of `value`. I would pass an `active` prop to this component, then it doesn't have to know about what value looks like

Reply:
Done in 2dde2b382c91168a00c13eedb705ffa2df7ffc80.